airbyte-webapp/package.json

```diff
@@ -49,6 +50,7 @@
     "rehype-urls": "^1.1.1",
     "remark-frontmatter": "^4.0.1",
     "remark-gfm": "^3.0.0",
+    "rxjs": "^7.5.5",
```

Comment:
Very bold introduction of rxjs. Did you consider alternatives of implementing the similar behavior without observables?

Reply:
Not really. I think Observables are a very well established pattern in the JS world and on their way into the standard https://github.com/tc39/proposal-observable, given that they fulfill the need for a clear API of streaming events, thus I think if we have the need for that use-case we should use it as the most standard implementation, rather than trying to "work around" them by solving it in another way (e.g. I could have tried to pull the event emitter to a higher level).

Reply:
I'm neutral on their inclusion. Happy to try it out.